org.gnome.Raider.yml

```diff
@@ -0,0 +1,22 @@
+app-id: org.gnome.Raider
```

Comment:
the app-id in the source is `org.gnome.raider`. This should match.

Also is this officially GNOME?